data-serving/data-service/src/index.ts

```diff
@@ -40,6 +42,20 @@ if (!deployment_envs.includes(env.SERVICE_ENV)) {
 
 // Express configuration.
 app.set('port', env.PORT);
+
+// log all non-200 responses: this needs to come _before_ any middleware or routers
+app.use(expressWinston.logger({
+    transports: [
+        new winston.transports.Console(),
+    ],
+    format: winston.format.json(),
+    /* don't log user info. We don't get user cookies or passwords in this service, so it's just
+     * belt-and-braces to ensure we don't log the API key if it was forwarded from the curator service.
+     */
+    headerBlacklist: ['X-Api-Key'],
```

Comment:
this is case-insensitive, right?

Reply:
[yes](https://github.com/bithavoc/express-winston/blob/main/index.js#L120) :)